doc/nrf/ug_bt_mesh_architecture.rst

```diff
@@ -53,16 +53,16 @@ The following process takes place at this stage:
   #. The model includes an on/off message in a publishing packet with an opcode and sends it to the access layer.
   #. The access layer fetches the necessary publish parameters, like destination address, encryption keys and time to live value (TTL value), and passes the packet to the transport layer, the highest of the core layers.
   #. The transport layer then encrypts the message with the selected application key, and splits the message into segments if necessary.
-     Each segment is passed to the network layer, which attaches a network header with a sequence number and encrypts the packet with the network key before passing it to the |BLE| controller.
-  #. The |BLE| controller includes the network message in an advertisement packet, and schedules a time slot for the packet to be broadcasted.
+     Each segment is passed to the network layer, which attaches a network header with a sequence number and encrypts the packet with the network key before passing it to the Bluetooth LE controller.
+  #. The Bluetooth LE controller includes the network message in an advertisement packet, and schedules a time slot for the packet to be broadcasted.
```

Comment:
@mia-ko , I guess we need SoftDevice Controller here, no?

Reply:
No, Bluetooth mesh in NCS does not use SoftDevice Controller.

Reply:
Should it still be Bluetooth LE **C**ontroller though?

Reply:
Good question. I have seen both BLE controller and BLE Controller be used in Zephyr doc... Who decides? :)

Reply:
The deal is, as Zephyr used both forms, Trond and I agreed to use BLE controller (due to our wish to use as much sentence case as possible). But, what is of more importance is which form is more widely used in NCS. If we mostly use Controller, then that is what we should use here too. And since it is called SoftDevice Controller, we should probably align with that and say BLE Controller.
So go for BLE Controller, if you agree. Checked with Trond, he agrees with me.

Reply:
Updated to Bluetooth LE Controller.